Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -1476,10 +1479,10 @@ private void HandleFalloffPerFrame()
                         }
                         else
                         {
-                            avgVelocity *= velocityDampen;
```

Comment:
nit, but seems like some whitespaces changes here that could be reverted

Reply:
Fixed!